### PR TITLE
[2/2] Implement RTK routes for node result and content routes

### PR DESCRIPTION
**Pull request by likawind**
## Describe your changes and why you are making these changes
This PR implements RTK routes for a few node results and content related routes.

We also added a few improvements in codebase:
* Moved all v2 typescript handlers to a subfolder so that it's clear which ones are from V2 and which ones are from v1
* Ensures all v2 typescript handlers has correct .go file name in their starting comments
* removed unnecessary assignments for `transformErrorResp`
* removed unnecessary imports in `index.ts`
## Loom demo (if any)
Verified existing features keeps working with a full rebuild.

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/golang/cmd/server/handler/v2/dag_get.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // This file should map directly to
-// src/ui/common/src/handlers/DagGet.tsx
+// src/ui/common/src/handlers/v2/DagGet.tsx
```
Comment (agiron123) [inline src/golang/cmd/server/handler/v2/dag_get.go:18]: Curious, when you run godoc, does this actually link to the file?
Or are IDEs able to to follow this and go directly there?

File: src/ui/common/src/handlers/parameters/ApiKey.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export type APIKeyParameter = {
```
Comment (agiron123) [inline src/ui/common/src/handlers/parameters/ApiKey.ts:1]: Should these all be in separate files, or should they all be combined into one index.ts file inside of the parameters folder?

**Review by agiron123 — APPROVED** [review]
LGTM, left small comments but this can be merged after they are addressed.

File: src/ui/common/src/handlers/parameters/ApiKey.ts (hunk 2 above)
Comment (likawind) [reply src/ui/common/src/handlers/parameters/ApiKey.ts:1]: I like the idea of combining them. I think we can split them into `header` `path` and `query` to clearly indicate where the parameter applies

File: src/golang/cmd/server/handler/v2/dag_get.go (hunk 1 above)
Comment (likawind) [reply src/golang/cmd/server/handler/v2/dag_get.go:18]: nope there's no automated navigation :(

**Pull request merged**